### PR TITLE
core metadata updates - round 2

### PR DESCRIPTION
**Pull request by mairaw**
I've moved more metadata to the global settings so making round 2 for the core folder

Related to #5062

### CODE REVIEW

**Review by JRAlexander — COMMENTED** [review]
Saw some what seem like inconsistencies

File: docs/core/docker/building-net-docker-images.md [hunk 1]
```diff
@@ -5,13 +5,8 @@ author: jralexander
 ms.author: johalex
 ms.date: 11/06/2017
 ms.topic: tutorial
-ms.prod: dotnet-core
-ms.technology: dotnet-docker
```
Comment (JRAlexander) [inline docs/core/docker/building-net-docker-images.md]: shouldn't ms.technology stay here?

File: docs/core/docker/docker-basics-dotnet-core.md [hunk 2]
```diff
@@ -5,13 +5,8 @@ author: jralexander
 ms.author: johalex
 ms.date: 11/06/2017
 ms.topic: tutorial
-ms.prod: dotnet-core
-ms.technology: dotnet-docker
```
Comment (JRAlexander) [inline docs/core/docker/docker-basics-dotnet-core.md]: here too

File: docs/core/docker/index.md [hunk 3]
```diff
@@ -4,12 +4,6 @@ description: Learn about using Docker with .NET Core using these fun tutorials.
 author: jralexander
 ms.author: johalex
 ms.date: 01/10/2018
-ms.topic: conceptual
-ms.prod: dotnet-core
-ms.technology: dotnet-docker
```
Comment (JRAlexander) [inline docs/core/docker/index.md]: Here too?

File: docs/core/docker/intro-net-docker.md [hunk 4]
```diff
@@ -4,14 +4,8 @@ description: Understanding Docker and .NET Core
 author: jralexander
 ms.author: johalex
 ms.date: 11/06/2017
-ms.topic: conceptual
-ms.prod: dotnet-core
-ms.technology: dotnet-docker
```
Comment (JRAlexander) [inline docs/core/docker/intro-net-docker.md]: Here

File: docs/core/sdk.md [hunk 5]
```diff
@@ -4,12 +4,7 @@ description: Find out about the .NET Core SDK, which is a set of libraries and t
 author: blackdwarf
 ms.author: mairaw
 ms.date: 06/20/2016
-ms.topic: conceptual
-ms.prod: dotnet-core
 ms.technology: dotnet-cli
```
Comment (JRAlexander) [inline docs/core/sdk.md]: You kept this ms.technology.

File: docs/core/testing/unit-testing-fsharp-with-mstest.md [hunk 6]
```diff
@@ -4,12 +4,8 @@ description: Learn unit test concepts for F# in .NET Core through an interactive
 author: billwagner
 ms.author: wiwagn
 ms.date: 08/30/2017
-ms.topic: conceptual
 dev_langs: 
   - "fsharp"
```
Comment (JRAlexander) [inline docs/core/testing/unit-testing-fsharp-with-mstest.md]: Left this devlang for F#. but the next file you remove it

File: docs/core/testing/unit-testing-fsharp-with-nunit.md [hunk 7]
```diff
@@ -3,12 +3,8 @@ title: Unit testing F# libraries in .NET Core using dotnet test and NUnit
...
 author: rprouse
 ms.date: 12/01/2017
-ms.topic: conceptual
-dev_langs:
- - "fsharp"
-ms.prod: dotnet-core
-ms.workload: 
-  - dotnetcore
+dev_langs: 
+  - "fsharp"
```
Comment (JRAlexander) [inline docs/core/testing/unit-testing-fsharp-with-nunit.md]: Added dev_langs back?

File: docs/core/testing/unit-testing-visual-basic-with-dotnet-test.md [hunk 8]
```diff
@@ -4,12 +4,8 @@ description: Learn unit test concepts in .NET Core through an interactive experi
 author: billwagner
 ms.author: wiwagn
 ms.date: 09/01/2017
-ms.topic: conceptual
 dev_langs: 
   - "vb"
```
Comment (JRAlexander) [inline docs/core/testing/unit-testing-visual-basic-with-dotnet-test.md]: Kept this one

File: docs/core/tutorials/vb-library-with-visual-studio.md [hunk 9]
```diff
@@ -4,13 +4,8 @@ description: Learn how to build a class library written in Visual Basic using Vi
 author: rpetrusha
 ms.author: ronpet
 ms.date: 08/07/2017
-ms.topic: conceptual
-ms.prod: dotnet-core
-ms.technology: devlang-vb
 dev_langs: 
   - "vb"
```
Comment (JRAlexander) [inline docs/core/tutorials/vb-library-with-visual-studio.md]: Should this one stay?

File: docs/core/tutorials/vb-with-visual-studio.md [hunk 10]
```diff
@@ -4,13 +4,8 @@ description: Learn how to build a simple .NET Core console application with Visu
 author: rpetrusha
 ms.author: ronpet
 ms.date: 08/07/2017
-ms.topic: conceptual
-ms.prod: dotnet-core
-ms.technology: devlang-vb
 dev_langs: 
   - "vb"
```
Comment (JRAlexander) [inline docs/core/tutorials/vb-with-visual-studio.md]: Should this one stay?

File: docs/core/tutorials/using-with-xplat-cli.md [hunk 11]
```diff
@@ -4,12 +4,7 @@ description: A step-by-step tutorial showing how to get started with .NET Core o
 author: cartermp
 ms.author: mairaw
 ms.date: 03/08/2017
-ms.topic: get-started-article
-ms.prod: dotnet-core
 ms.technology: dotnet-cli
```
Comment (JRAlexander) [inline docs/core/tutorials/using-with-xplat-cli.md]: Should this stay?

File: docs/core/deploying/creating-nuget-packages.md [hunk 12]
```diff
@@ -4,12 +4,7 @@ description: Learn how to create a NuGet package with the 'dotnet pack' command.
 author: cartermp
 ms.author: mairaw
 ms.date: 06/20/2016
-ms.topic: conceptual
-ms.prod: dotnet-core
 ms.technology: dotnet-cli
```
Comment (JRAlexander) [inline docs/core/deploying/creating-nuget-packages.md]: Should this one stay?

**Review by JRAlexander — COMMENTED** [review]
Looks good except for a few items.

**Review by rpetrusha — APPROVED** [review]
Generally LGTM, @JMAlexander left a number of comments, and I left one.

File: docs/core/deploying/creating-nuget-packages.md (hunk 12 above)
Comment (mairaw) [reply docs/core/deploying/creating-nuget-packages.md]: Yes, they are not on the global settings so I left them. @rpetrusha do you think these should continue to be tagged as CLI?

File: docs/core/additional-tools/index.md [hunk 13]
```diff
@@ -5,11 +5,7 @@ author: mlacouture
 manager: wpickett
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/additional-tools/index.md]: Good catch @rpetrusha!

File: docs/core/sdk.md (hunk 5 above)
Comment (mairaw) [reply docs/core/sdk.md]: Same reason. Not under tools where I set the ms.technology globally.

File: docs/core/testing/unit-testing-fsharp-with-nunit.md (hunk 7 above)
Comment (mairaw) [reply docs/core/testing/unit-testing-fsharp-with-nunit.md]: I just removed ms.topic and the values below dev_langs. It's just the diff that is not aligning these correctly @JRAlexander.

File: docs/core/testing/unit-testing-fsharp-with-mstest.md (hunk 6 above)
Comment (mairaw) [reply docs/core/testing/unit-testing-fsharp-with-mstest.md]: I didn't touch (or not supposed to touch) any of dev_langs.

File: docs/core/tools/custom-templates.md [hunk 14]
```diff
@@ -4,12 +4,6 @@ description: Learn about custom templates for any type of .NET project or files.
 author: guardrex
 ms.author: mairaw
 ms.date: 08/11/2017
-ms.topic: conceptual
-ms.prod: dotnet-core
-ms.technology: dotnet-cli
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/tools/custom-templates.md]: yes, if you see on preview, this file should have the value coming from the global setting.

**Review by mairaw — COMMENTED** [review]
@JRAlexander I think I've addressed all your questions, but please let me know if there's something still not clear. Thanks!

**Pull request merged**